robottelo/ui/role.py

```diff
@@ -85,3 +85,29 @@ def add_permission(self, role_name, resource_type=None,
             self.click(tab_locators['roles.tab_location'])
             self.configure_entity(location, FILTER['filter_loc'])
         self.click(common_locators['submit'])
+
+    def get_resources(self, role_name):
```

Comment:
Can see one flaw in these 2 `get_*` methods - they only work in case of <=10 resources, as pagination isn't handled. Not sure if we should handle that, as your tests are the only ones which uses these helpers, but i think it worth mentioning in comment with fixme/todo to avoid 'surprises' later (e.g. if one tries to use them with 'Viewer' role, there's 49 entries as for now).